### DISCUSSION TITLE
Get back default chat channels?

### DISCUSSION THREAD

**Discussion by matt-h** [zed-industries/zed#10729] [Feature Requests]
When first launching Zed, in the chat channel panel there was a whole list of public channels to join.

I removed those when testing thing out, and I see no way to get them back. Is there a way to restore those channels?

**Comment by JosephTLyons** [comment]
Hey @matt-h - are you referring to the Zed team's channel tree?

https://zed.dev/channel/zed-283

**Reply by matt-h** [reply]
Thanks! Yes, I meant that channel.

Running `zed://channel/zed-283` in the command pallet in zed command pallet got it back.
